keyboards/squarebuilt/keymaps/default/keymap.c

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2019 LanceMercado
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#define _QWERTY 0
```

Comment:
I'd recommend using an `enum` in place of Lines 17-20 for your layer indexes. Right now, your `_ADJUST` is at 16, which means the compiler will insert blank layers at indexes 3 through 15, which will increase the compiled size of your firmware, for no benefit in functionality.

```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```